Errata.md

```diff
@@ -1,11 +1,26 @@
 # Errata
 
-Errata (excluding minor typos) for the book.
 
 # 1st Printing
 
 * pg. xvi bit "and may require a couple **read troughs**" should be "bit rapid, and may require a couple **read-throughs**" (Thanks John M. Shea)
 
-* pg. 262, In Figure 9.1. "Model **Compasion**" should be "Model **Comparison**" (Thanks Ben Vincent)
+* pg. xvi "For a reference on Python, or how to setup the computation environment needed for this book, go to README.md in Github to understand how to setup a code environment" 
+  on the online version should be: "For a reference on how to setup the computation environment needed for this book, read environment installation."
```

Comment:
Should the online version be listed here? Maybe it is cleaner to show errata for the print version (with the online version being the corrected version).

Reply:
I only make a difference here because the online and printed version are different. But yes we can list only the printed version